docs/reference/search/profile.asciidoc

```diff
@@ -673,13 +704,15 @@ The API returns the following result:
             ]
           }
         ],
-        "aggregations": [...] <1>
+        "aggregations": [...], <1>
+        "fetch": {...}
       }
     ]
   }
 }
 --------------------------------------------------
 // TESTRESPONSE[s/"aggregations": \[\.\.\.\]/"aggregations": $body.$_path/]
+// TESTRESPONSE[s/"fetch": \{\.\.\.\}/"fetch": $body.$_path/]
```

Comment:
If we like this PR I'll update the docs to add a section describing the fetch profile. The guts of a profile are in a kind of grey area where they are quite part of the public API but folks can look at them and kibana even visualizes then but we will change them when the underlying implementations change. Its a weird place.